### ISSUE TITLE
Will this code compile using Android Studio V3.0.1 as every time I produce an APK file, I get a Parse Error on my phone when trying to install it.

### ISSUE THREAD

**Issue by Alzibiff** [MilosKozak/AndroidAPS#735]
Reporting bugs
--------------
- Note the precise time the problem occurred and describe the circumstances and steps that caused
  the problem
- Note the Build version (found in the About dialog in the app, when pressing the three dots in the
  upper-right corner).
- Obtain the app's log files, which can be found on the phone in
  _/storage/emulated/0/Android/data/info.nightscout.androidaps/_
  See https://github.com/MilosKozak/AndroidAPS/wiki/Accessing-logfiles
- Open an issue at https://github.com/MilosKozak/AndroidAPS/issues/new

**Comment by AdrianLxM** [comment]
Does it work in the emulator?

**Comment by Alzibiff** [comment]
I'll be honest - The reason I posted this as an issue was because I have been struggling to compile the code which will actually install. I am totally new to Android Studio and have been using the instructions on the Wiki for this site in order to produce the APK file. I am using a phone running Android 6 - the error is probably down to me and the purpose of the issue was to try and confirm that there are no problems when using the latest version of Studio (which I installed yesterday) on a Windows 10 machine.

I have not been able to get the emulator to run - advised that it needs something changing in the BIOS of my computer and I would prefer to try and solve the problem without this.
